### ISSUE TITLE
Please release indentation fixes

### ISSUE THREAD

**Issue by seanarnold** [thoughtbot/factory_girl_rails#183]
When generating a factory with multiple attributes the indentation is wrong.

```
FactoryGirl.define do
  factory :product do
    name "MyString"
short_name false
  end

end
```

This was fixed way back in 2014, but still hasn't been released (https://github.com/thoughtbot/factory_girl_rails/commit/3488c767c76d6fe3d187c788ec158cf3d9649798). Can we release a `4.5.1` that includes these please.

**Comment by stephenbaidu** [comment]
+1

**Comment by seanarnold** [comment]
@joshuaclayton or anyone else?

**Comment by jferris** [comment]
I just released 4.6.0 which includes these changes. Sorry for the delay.

**Comment by stephenbaidu** [comment]
@jferris Thanks.
